platform/ui/src/views/StudyList/components/Header.js

```diff
@@ -12,7 +12,7 @@ function Header({ appLogo = OHIFLogo(), children, t }) {
   };
 
   return (
-    <div className="bg-custom-navy flex flex-row justify-between px-3 py-1 text-white sticky top-0 z-10 border-b-4 border-black">
+    <NavBar className="justify-between border-b-4 border-black">
```

Comment:
A tiny bit torn. @rodrigolabs, should border be added inside our outside of component? are there any scenarios where you wouldn't want border?

I do agree with `justify-between` being external.